pkg/reconciler/serverlessservice/resources/services.go

```diff
@@ -155,6 +155,14 @@ func MakePrivateService(sks *v1alpha1.ServerlessService, selector map[string]str
 				Protocol:   corev1.ProtocolTCP,
 				Port:       networking.QueueAdminPort,
 				TargetPort: intstr.FromInt(networking.QueueAdminPort),
+			}, {
```

Comment:
Do we want to guard this on the feature flag?

Reply:
That's the question I asked in the description as well. I don't know how expensive the extra port is tbh. especially compared to the "expense" of threading the config through here which will complicate code and tests.